File: 0_Trabalho_Pratico_Padaria/Bakehouse/Bakehouse.Domain/Entities/Product.cs
using System.Collections.Generic;

namespace Bakehouse.Domain.Entities
{
    public class Product : BaseEntities
    {
        public int Id { get; set; }
        public string BarCode { get; set; }
        public string Description { get; set; }
        public double Quantity { get; set; }
        public double ValueUnitary { get; set; }
        public double MinQuantity { get; set; }

        
        public virtual UnitOfMeasurement UnitOfMeasurement { get; set; }
        public int UnitOfMeasurementId { get; set; }
        public virtual Category Category { get; set; }
        public int CategoryId { get; set; }
        public virtual ICollection<OrderPadItem> OrderPadItems { get; set; }
    }
}

File: 0_Trabalho_Pratico_Padaria/BakehouseJava/src/com/bakehouse/domain/Product.java
package com.bakehouse.domain;

public class Product {
    private int id;
    private String barCode;
    private String description;
    private int quantity;
    private double valueUnitary;
    private int minQuantity;
    
    private UnitOfMeasurement unitOfMeasurement;
    private Category category;
}